jobs/pending_payments.ts

```diff
@@ -55,12 +57,13 @@ exports.attemptPendingPayments = async (bot: Telegraf<MainContext>): Promise<voi
         order.status = 'SUCCESS';
         order.routing_fee = payment.fee;
         pending.paid = true;
-        pending.paid_at = new Date().toISOString();
+        pending.paid_at = new Date();
         // We add a new completed trade for the buyer
         buyerUser.trades_completed++;
         await buyerUser.save();
         // We add a new completed trade for the seller
         const sellerUser = await User.findOne({ _id: order.seller_id });
+        if (sellerUser === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program